### PR TITLE
NMS-6118 : Add additional relative time periods to choose from.

### PR DESCRIPTION
**Pull request by roskens**
NMS-6118 : Add additional relative time periods to choose from.
 * Last Hour
 * Last 2 Hours
 * Last 4 Hours
 * Last 8 Hours
 * Last 12 Hours

* JIRA: http://issues.opennms.org/browse/NMS-6118

### CODE REVIEW

File: opennms-web-api/src/main/java/org/opennms/web/svclayer/model/RelativeTimePeriod.java [hunk 1]
```diff
@@ -48,6 +48,16 @@
     
     static {
         s_defaultPeriods = new RelativeTimePeriod[] {
+                new RelativeTimePeriod("1_hour", "Last Hour", Calendar.HOUR,
```
Comment (mvrueden) [inline opennms-web-api/src/main/java/org/opennms/web/svclayer/model/RelativeTimePeriod.java]: can we make them only use one line? This looks weird. Can we also name them `last_1_hour` or something like that?

**Comment by mvrueden** [comment]
This was originally targeted to be merged to `master`. Master should be protected and is for releases only. I changed the branch to be `develop` instead. Thank you (-:

File: opennms-web-api/src/main/java/org/opennms/web/svclayer/model/RelativeTimePeriod.java (hunk 1 above)
Comment (roskens) [inline opennms-web-api/src/main/java/org/opennms/web/svclayer/model/RelativeTimePeriod.java]: Do you want me to change the existing ones to one line also? Looks like the rest of the file is formatted for max 80 columns.

Comment (mvrueden) [inline opennms-web-api/src/main/java/org/opennms/web/svclayer/model/RelativeTimePeriod.java]: It is up to you (-: I would appreciate it (-:

**Comment by roskens** [comment]
Rebase'd against develop.
Changed lines to single per RelativeTimePeriod

**Comment by j-white** [comment]
Looks good to me. Pushed to `pr/jira/NMS-6118`. Waiting on Bamboo.

**Pull request merged**